CMakeModules/LlsLint.cmake

```diff
@@ -0,0 +1,35 @@
+function(init_check_lua_style)
+    if(OE_CHECK_LUA_STYLE)
+        unset(OE_LUALANGUAGESERVER_COMMAND CACHE)
+        unset(OE_LLS_CHECK_COMMAND CACHE)
+        find_program(OE_LUALANGUAGESERVER_COMMAND lua-language-server)
+        set(OE_LLS_CHECK_COMMAND "${PROJECT_SOURCE_DIR}/lls-check.py" CACHE FILEPATH "Lua Language Server check script")
+        if(OE_LUALANGUAGESERVER_COMMAND)
+            message("Lua Language Server Found")
+        else()
+            message("Could not find Lua Language Server")
+        endif()
+    endif()
+endfunction()
+
+function(source_check_lua_style TARGET TARGET_SOURCES)
```

Comment:
It feels weird that we're not using the TARGET_SOURCES in this function.

Reply:
Ah right I should have removed them entirely actually. The sources are not used because, unfortunately, luaLS looks directly at the specified folder.

The fix here is to not expect TARGET_SOURCES at all.

Reply:
Yup! It works without them as well 😃